### PR TITLE
update admiral links to last release

### PR DESCRIPTION
**Pull request by dgrassellyb**
Update links to point to the new admiral multi-version docs page.

### CODE REVIEW

**Comment by bms63** [comment]
@bundfussr @pharmaverse/admiralonco please check out this is related to https://github.com/pharmaverse/admiralci/issues/22

File: .github/ISSUE_TEMPLATE/05_onboard.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ body:
         - label: Given a tour of Github from a Core member
         - label: Understand how to Create Issues and do a Pull Request
         - label: Understand the Programming Strategy
-        - label: Read and understand [Developer Guides Articles](https://pharmaverse.github.io/admiral/index.html)
+        - label: Read and understand [Developer Guides Articles](https://pharmaverse.github.io/admiral/cran-release/index.html)
```
Comment (bundfussr) [inline .github/ISSUE_TEMPLATE/05_onboard.yml:21]: The developer guides were moved to admiraldev. Thus it should link to https://pharmaverse.github.io/admiraldev/devel/articles/index.html.

File: .github/ISSUE_TEMPLATE/05_onboard.yml [hunk 2]
```diff
@@ -33,7 +33,7 @@ body:
...
```
Comment (bundfussr) [inline .github/ISSUE_TEMPLATE/05_onboard.yml:36]: As above

File: README.md [hunk 3]
```diff
@@ -40,7 +52,7 @@ sensitivity analyses and questionnaires.
 
 ## References and Documentation
 
-* Please refer to the [`{admiral}` References and Documentation](https://pharmaverse.github.io/admiral/index.html#references-and-documentation)
+* Please refer to the [{admiral} References and Documentation](https://pharmaverse.github.io/admiral/main/index.html#references-and-documentation)
```
Comment (bundfussr) [inline README.md:55]: Should it be `cran-release` instead of `main`?

**Review by bundfussr — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (dgrassellyb) [reply README.md:55]: @cicdguy should we change this one also ? (If yes I will also update the .Rmd admiral_homepage but just let me know !)

Comment (dgrassellyb) [reply README.md:55]: Okay I am just fixing it :)

**Review by bundfussr — APPROVED** [review]

**Comment by bundfussr** [comment]
@cicdguy , I have merged it.

By the way, did you have a chance to look into the failing R-CMD check for R 4.0.

**Pull request merged**